awestruct.gemspec

```diff
@@ -34,7 +34,7 @@ Haml and Markdown filters are touchy things. Redcarpet or Rdiscount work well if
 Compass and sass are no longer hard dependencies. You'll need too add them on your own should you want them. We also should be able to work with sassc.
   EOS
 
-  s.add_dependency 'haml', '~> 4.0', '>= 4.0.5'
+  s.add_dependency 'haml', '>= 4.0', '< 6.0'
```

Comment:
We need at least 4.0.5 instead of just the 4.x branch.

Reply:
Ah right! I'll change it

Reply:
Done